WalletWasabi/Crypto/ZeroKnowledge/NonInteractive/IStatement.cs

```diff
@@ -0,0 +1,46 @@
+using WalletWasabi.Crypto;
+using System.Collections.Generic;
+
+namespace WalletWasabi.Crypto.ZeroKnowledge.NonInteractive
+{
+	/// <summary>
+	/// This delegate represents the second phase of proof verification with <see
+	/// cref="IStatement"/>.
+	/// </summary>
+	public delegate VerifyResponse VerifierCommitToNonces(Proof proof);
+
+	/// <summary>
+	/// This delegate represents the third phase of proof verification with <see
+	/// cref="IStatement"/>.
+	/// </summary>
+	public delegate bool VerifyResponse();
+
+	/// <summary>
+	/// <see cref="IStatement"/> splits verification of a non interactive proof using
+	/// into three phases corresponding to those of <see cref="IKnowledge"/>:
+	/// <list type="number">
+
```

Comment:
This blank line is invalid in xmldoc, I believe.

Reply:
it seems i accidentally pasted those lines right before comitting